modules/mod_speasyimagegallery/helper.php

```diff
@@ -101,4 +101,31 @@ private static function getItemID() {
 		return;
 	}
 
+
+	/**
+	 * Retrieves the album description based on module parameters.
+	 *
+	 * @param  object 		$params -- The module parameters.
+	 * @return string|null 			-- The album description, or null if not found or not configured to display.
+	 * @since  2.0.5
+	 */
+	public static function getAlbumDescription($params)
+	{
+		if (!$params->get('show_album_desc',0)) {
```

Comment:
use space after comma

```suggestion
		if (!$params->get('show_album_desc', 0)) {
```